src/components/AddComment.jsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+const AddComment = ({ article_id, setComments }) => {
+  const [newAuthor, setNewAuthor] = useState("");
+  const [newBody, setNewbody] = useState("")
+    const [error, setErr] = useState(null)
+  
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newComment = {
+        author: newAuthor,
+        body: newBody,
+        username: newAuthor,
```

Comment:
Rarther than a text input box for the username (difficult to know which users exist) please could you hardcode a username here and remove the username input. 

This way you can modify it to use a login much easier if you get there 👍 

*Same for the author key on this object